docs/01_key-concepts/04_net.md

```diff
@@ -1,10 +1,8 @@
 ---
 content_title: NET as system resource
-link_text: NET as system resource
 ---
 
 NET, as CPU and RAM, is a very important system resource in EOSIO-based blockchains. NET is measured by the byte size of the transactions saved in the blockchain database and it is referred to as `net bandwidth` on the cleos get account command result. When a transaction is executed by the blockchain it consumes CPU and NET, therefore sufficient NET must be staked in order for transactions to complete.
 
 For more details about EOSIO staking refer to the following:
-* [Staking Mechanism](https://developers.eos.io/welcome/latest/overview/technical_features#staking-mechanism).
```

Comment:
We have a working link now:

\* [Staking Mechanism]\(https://docs.eosnetwork.com/docs/latest/overview/technical_features#staking-mechanism).

Please add back.